src/kernel/src/fs/perm.rs

```diff
@@ -136,8 +136,26 @@ impl From<Mode> for u32 {
 
 bitflags! {
     /// An implementation of `accmode_t`.
+    /// Some of the constants are unused, but we include them for completeness.
     pub struct Access: u32 {
-        const EXEC = 000000000100; // VEXEC
+        const EXEC             = 0o00000000100; // VEXEC
+        const WRITE            = 0o00000000200; // VWRITE
+        const READ             = 0o00000000400; // VREAD
+        const ADMIN            = 0o00000010000; // VADMIN
+        const APPEND           = 0o00000020000; // VAPPEND
+        const EXPLICIT_DENY    = 0o00000100000; // VEXPLICIT_DENY
+        const READ_NAMES_ATTR  = 0o00000200000; // VREAD_NAMES_ATTR
+        const DELETE_CHILD     = 0o00001000000; // VDELETE_CHILD
+        const READ_ATTRIBUTES  = 0o00002000000; // VREAD_ATTRIBUTES
+        const WRITE_ATTRIBUTES = 0o00004000000; // VWRITE_ATTRIBUTES
+        const DELETE           = 0o00010000000; // VDELETE_CHILD
+        const READ_ACL         = 0o00020000000; // VREAD_ACL
+        const WRITE_ACL        = 0o00040000000; // VWRITE_ACL
+        const WRITE_OWNER      = 0o00100000000; // VWRITE_OWNER
+        const SYNCHRONIZE      = 0o00200000000; // VSYNCHRONIZE
+
+        const ADMIN_PERMS = Self::ADMIN.bits() | Self::WRITE_ATTRIBUTES.bits() | Self::WRITE_ACL.bits() | Self::WRITE_OWNER.bits();
+        const STAT_PERMS = Self::READ_ATTRIBUTES.bits() | Self::READ_ACL.bits();
```

Comment:
Have you verified these with the PS4? If not better to remove it since the PS4 might be different. We already have this issue with `mmap` before that the value is not the same as FreeBSD.

Reply:
Most of these are used in unixify_access and I doubt that. I can remove the rest if you want

Reply:
You can use `unixify_access` on the PS4 to verified if those values are actually the same. Then remove all of non-verified ones.

Reply:
If this:
<img width="215" alt="image" src="https://github.com/obhq/obliteration/assets/48620541/d1a425df-58ab-40c7-b3c9-9987e08f2a69">
is equivalent to access.intersects(Access::EXPLICIT_DENY), then it is correct. I removed the two unused constants